src/util.ts

```diff
@@ -27,7 +27,8 @@ export const getCondaArch = () => {
     'linux-x64': 'linux-64',
     'linux-arm64': 'linux-aarch64',
     'linux-ppc64': 'linux-ppc64le',
-    'win32-x64': 'win-64'
+    'win32-x64': 'win-64',
+    'win32-arm64': 'win-arm64'
```

Comment:
Could you add a unit test in .github/workflows/test.yml as well?

Reply:
Added `windows-11-arm` (and also `ubuntu-24.04-arm`) runners to the matrix build for the "bash" job at commit 9d33c874eefd6a01ba6433280a50c391c6c44dd0, hopefully that's ok?